### DISCUSSION TITLE
Hire Me (April 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#64034] [Show and tell]
# What is this?

This discussion is for Next.js developers looking for work.

## Candidates

Share your information if you are looking for work.

Please use this format:

> Location:
> Remote:
> Willing to relocate:
> Technologies:
> Resume/CV:
> Website:
> Email:

Good luck with interviewing!

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍

If you're a company looking to hire, please use #64033 instead.

We've had yet another [success story](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript and Mobile Applications  (Check [GitHub](https://github.com/jensen))
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://jensen.social
Email: nextjs@jensen.social

Previously: EA Sports
Recently shipped: https://alicent.ai/, https://rlife.app

In 2021/2022 I built over **20** web applications which can be found on my [GitHub](https://github.com/jensen) profile.

**I am only looking for contract work at this time.**

**Comment by Itsnotaka** [comment]
Location: Taiwan & Hong Kong (Permanent) / US (F-1)
Remote: Yes
Willing to relocate: Yes
Technologies:JavaScript, TypeScript, React/Next.js, Node.js, tRPC, Tailwind CSS, SQL, Websocket, Electron, Vercel, AWS
Resume/CV: [https://cv.nameisdaniel.com](https://cv.nameisdaniel.com)
Website: [https://nameisdaniel.com](https://nameisdaniel.com)
Email: [nameisdaniel@flowapp.so](mailto:nameisdaniel@flowapp.so) / [daniel.fu90@gmail.com](mailto:daniel.fu90@gmail.com)

**Comment by Phoenix-Alpha** [comment]
...

**Comment by evergreenx** [comment]
Location: Lagos , Nigeria 
Remote: yes
Willing to relocate: yes
Technologies:JavaScript, TypeScript, React/Next.js, Node.js, tRPC, Tailwind CSS, Storybook

Resume/CV:https://docs.google.com/document/d/1bz5DpmVakuQ_e_ULolTAWl5GVndHr2RsZhxM6SEpQJ4/edit?usp=drivesdk

Website: https://idoevergreen.me/
Email: dicksonevergreenido@gmail.com

**Comment by GetPsyched** [comment]
Location: India
Remote: Yes
Willing to relocate: Yes (within Asia & Europe)
Technologies:
- Languages: TypeScript, Python, Go, Rust, Nix
- Frameworks/Libraries: NextJS, React, Svelte, HTMX, TailwindCSS, NodeJS, Prisma, Drizzle, ExpressJS
- Databases: NoSQL, MySQL, PostgreSQL

Resume/CV: https://getpsyched.dev/resume
LinkedIn: https://www.linkedin.com/in/priyanshutr
Email: `jobs@getpsyched.dev`

**Comment by lili21** [comment]
Location: China, ShangHai
Remote: Yes
Willing to relocate: Yes
Technologies: React, Remix, NextJS, Web Performance, Tailwind, NodeJS
Resume/CV: I can sent it via email.
Website: https://blog.lili21.me
Email: mark.li210812@gmail.com

**Comment by kralion** [comment]
Location: Perú
Remote: Yes
Willing to relocate: No
Technologies:

- Languages: TypeScript, Python, Go
- Frameworks/Libraries: NextJS, React, Nuxt, TailwindCSS, NodeJS, Prisma, tRPC, NestJS, ExpressJS
- Databases: NoSQL, MySQL, PostgreSQL, SQLite

Resume/CV: https://drive.google.com/file/d/1uWEIOPXvljmVd2ppGNZ5q4QldBlBUuGC/view
LinkedIn: https://www.linkedin.com/in/brayanpaucar
Email: brayanjoanpm@gmail.com

**Comment by nemanjam** [comment]
Location: Serbia, GMT +2
Remote: Yes
Willing to relocate: Maybe
Technologies: React, Next.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Cypress, Prisma, PostrgreSQL, MongoDB, Docker, C#/.NET
Resume/CV: Available on request, please send me an email
GitHub: https://github.com/nemanjam
LinkedIn: https://www.linkedin.com/in/nemanja-mitic
Email: nemanja.mitic.elfak [at] hotmail.com


I am a full stack developer with a Master's degree in Computer Science and 4 years of professional experience with JavaScript and C#/.NET. In the last few years my main focus has been the React/Next.js ecosystem, followed by Node.js. I am a JavaScript professional and a Python beginner interested to get more exposure to it. I have also worked with ASP.NET in the past which is not my primary focus at the moment. I have a basic understanding of DevOps tools.

A quick and easy way to assess my current skill level, the quality of my code and documentation, and my preferred tech stack (Next.js, TypeScript, React Query, Prisma, Postgres, TailwindCSS, Docker, Github Actions) is to check out one of my latest projects: https://github.com/nemanjam/nextjs-prisma-boilerplate .

I have experience working remotely for startups within globally distributed multicultural teams.

I am looking for a full-time remote position. We can discuss relocation after some time spent in the company. You can check out my other projects at https://github.com/nemanjam . I have detailed resume available, you can tell me about your company and project by email: nemanja.mitic.elfak [at] hotmail.com or by LinkedIn: https://www.linkedin.com/in/nemanja-mitic .

**Comment by lawlesx** [comment]
Location: India, Bengaluru
Remote: Yes
Willing to relocate: No
Technologies:
  - Languages - Typescript, Javascript, C++ and Python
  - Frameworks - Nextjs, Reactjs, Nodejs, Expressjs
  - Libraries - TailwindCSS, React Query, Redux, Framer motion, Prisma, Zod, Chakra UI, NextUi
  - Other - Vercel, GraphQl,  REST API, JWT Auth, Google Analytics

Resume/CV: https://drive.google.com/file/d/13CctzMWI8sa5c2hsMtieDu0AIyeFHpa3/view?usp=sharing
Website: https://lawlesx.vercel.app/
Email: aniruddhasil109@gmail.com

**What drives me** - Traditional websites often feel static and uninspiring. My goal is to inject life into them, crafting interfaces that are not only beautiful but also intuitive and engaging. I possess an uncanny ability to accurately recreate any 2D website element, ensuring a visually consistent and polished user experience. Breathing Life into Interfaces: Framer Motion is my playground. I take immense satisfaction in crafting smooth and captivating animations that elevate user engagement.

**Comment by Starko00** [comment]
Location: Europe - but able to remove anywhere 
Remote: Yes
Willing to relocate: Yes - Europe, USA, Australia and or New Zeland
Technologies: Next.js, PSQL, NodeJS, MongoDB, Traditional SQL and many more 
Resume/CV:[2024.pdf](https://github.com/vercel/next.js/files/14893550/2024.pdf)
Website: https://starko.one
Email: stefan@starko.me

**Comment by CantBeSubh** [comment]
Location: New Delhi, India(GMT +5.5hrs)
Remote: Yes
Willing to Relocate: Yes
Technologies: React, NextJS, Vue, NuxtJS, TailwindCSS, Prisma, MySQL, PostgreSQL, MongoDB, GraphQL, ThreeJS, Python, AWS, Docker
Resume: [Google Drive(resume.pdf)](https://drive.google.com/file/d/10NSHj-4Nu21l8kzcWxsYFnCLSZGeDS2t/view?usp=sharing)
Website: https://subhranshu.com/
Github: [github/CantBeSubh](https://github.com/CantBeSubh)
LinkedIn: [LinkedIn/subhranshu-pati](https://www.linkedin.com/in/subhranshu-pati/)
Email: [socials.subhranshu@gmail.com](mailto:socials.subhranshu@gmail.com)

**Comment by wangtiger317** [comment]
Location: China
Remote: Yes
Willing to Relocate: No
Technologies: React, NextJS, Vue, NuxtJS, TailwindCSS, Prisma, MySQL, PostgreSQL, MongoDB, GraphQL, ThreeJS, Python, Docker,Laravel,Shopify,Wordpress
Resume:(https://drive.google.com/file/d/1jJwGgA5JiPLt1LaG4SD0iCtz31AxDI8W/view?usp=drive_link)
Website: https://portofolio-0317.vercel.app/
Github:https://github.com/wangtiger317
Email: [phonglifeng@gmail.com]

**Reply by wangtiger317** [reply]
I have much experience

**Comment by Constantiner** [comment]
Location: Bar, Montenegro
Remote: Yes
Willing to relocate: Yes (North Americas)
Technologies: React, TypeScript, NextJS, TailwindCSS, SASS, Prisma, MySQL, PostgreSQL, Docker
Resume/CV: [Konstantin Kovalev](https://github.com/Constantiner/Constantiner/blob/main/Konstantin-Kovalev-Full-Stack.pdf)
Website: https://github.com/Constantiner
Email: constantiner@gmail.com

**Comment by josuembuyu** [comment]
Location: Kinshasa, Congo (GMT+1)
Remote: Yes
Willing to relocate: Yes
Technologies: 
-  Frontend: JS/TS | Vue/Nuxt.js | React/Next.js | Material UI | Tailwind CSS
- Backend: Express.js | Nest.js | Node.js
- Database: Postgres | MySQL | MongoDB
- Other: Redux, Recoil, Zustand | Prisma, TypeORM, Drizzle, Mongoose | GraphQL
- DevOps: AWS | Docker | Github Actions

Resume/CV: https://docs.google.com/document/d/1eBULY07GcTA3YV3Rq48MXv72AiItHNy0YqT4Effr5Rg/edit
LinkedIn : https://www.linkedin.com/in/josue-mbuyu/
Github: https://github.com/josuembuyu
Email: josuembuyu94@gmail.com

**Comment by MohammadElokour** [comment]
Status: US Citizen
Location: Amman, Jordan
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js 14 w/App Router, React + TypeScript, Javascript, ReactNative, Tailwind CSS, Material UI, Shadcn UI, Mobx, Recoil, RestAPI, Zod, headless CMS, Tanstack Table, and others.
Resume/CV: https://www.melokour.me/assets/melokour-resume-2fac3e0b.pdf
Website: https://melokour.me/
Email: [mfelokour@gmail.com](mailto:mfelokour@gmail.com)

Example of my work: [coretava.com](https://coretava.com/)

**Comment by benesvoj** [comment]
Location: Opava, Czech republic
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, React, Next.js, TypeScript, Tailwind CSS, Shadcn UI, Styled Components, MySQL, Postgres, SASS
Resume/CV: https://www.linkedin.com/in/benesvojtech/
Website: https://vojtechbenes.cz
Email: [email@vojtechbenes.cz](mailto:email@vojtechbenes.cz)

**Comment by KenzoBenzo** [comment]
**Location:** Brooklyn, NY
**Remote:** Yes, I'm open
**Willing to relocate:** No
**Technologies:** 
- `React` / `Next.js` (page & app router experience)
- Most CSS frameworks, lately `Tailwind CSS` or `Chakra UI`
- `TypeScript`
- Design system tooling like `CVA`, `Omlet`, `Radix-ui`, `React Aria`, `Headless UI`, `Storybook`/`Chromatic`

**Resume/CV:** https://makenna.dev/resume.pdf
**Website:** https://makenna.dev/
**Email:** makennasmutz@gmail.com

---
I'm motivated by building products that lean into wicked cool moments via stunning (micro) animations, polished design, and deeply thought-through processes. I thrive in autonomous, proactive environments, and am stoked to dive deep into new tech/tools and concepts.

I'm a well rounded engineer, I've solo-built several startups, with experience in Frontend, Backend, Product, Design, and even a little Marketing. I enjoy diving deep into the intersection of design and code through Design Engineering.

If my profile above sounds interesting, I'd love to chat.

**Comment by Anara5** [comment]
Location: Stockholm, Sweden
Remote: Yes
Willing to relocate: Yes (Scandinavia)

Technologies: 
- Frontend: JS/TS  | React/Next.js | Tailwind CSS, HTML, CSS;
- Backend: Express.js | Node.js;
- Database: Postgres | MySQL | NoSQL;
- Other: Java | Redux | GraphQL | Postman | REST API;
- DevOps: AWS | Docker | Github Actions.

LinkedIn: https://www.linkedin.com/in/anarazhunusova
Resume/CV: https://drive.google.com/file/d/14H_o08Hywv6_x4z6FY7uR7fjkUydiBQn/view?usp=sharing
Website: https://portfolio-1-anara5.vercel.app

**Comment by sagspot** [comment]
Location: Nairobi Kenya
Remote: Yes
Willing to relocate: Yes
Technologies: Nextjs, Nestjs, Expo
Resume/CV: https://cdn.sanity.io/files/7dl3hjrh/production/9bc32adac66ee759ecc309efd45107c820f00f20.pdf
Website: https://sagspot.co.ke
Email: hello@sagspot.co.ke

**Comment by Yashhh08** [comment]
Location: Mumbai, India
Remote: Yes
Willing to relocate: Yes
Technologies: MERN Stack, Next.js 14
Resume/CV: https://drive.google.com/file/d/1euyLJXIle0Odf0BOKKJKHTqCuUlIJZkf/view?usp=sharing
Website: https://yash-yerunkar.vercel.app/
Email: yashyerunkar8@gmail.com

**Comment by xjqiu28** [comment]
Location: New York City, New York
Remote: Yes
Willing to relocate: Yes
Technologies: MERN, Python, Java
Resume/CV: https://drive.google.com/file/d/1f2pYvCkdYLq5b93RaRgUbHsrvsPx7KXn/view?usp=sharing
Website: https://www.linkedin.com/in/xinjinqiu
Email: xinjin.qiu.1@gmail.com

**Comment by alfonsocartes** [comment]
Location: Europe (usually working US timezones also)
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, React Native, Expo, TypeScript, Javascript, React Native, Tailwind CSS, Node.js, Serverless and many more... **(I have over a decade of experience)**
Resume/CV: [https://www.linkedin.com/in/alfonsocartes](https://www.linkedin.com/in/alfonsocartes)
Website: [https://cartes.dev/]( https://cartes.dev/)
Email: [email](https://cartes.dev/contact)

**Comment by thatguynapster** [comment]
Location: Accra, Ghana
Remote: Yes
Willing to relocate: Yes
Technologies: MERN Stack, Next.js 14
Resume/CV: https://drive.google.com/file/d/1emCb_m9JjUmlANzhq_A7ohuSn61scMgV/view?usp=sharing
Website: https://thatguynapster.com
Email: [andrewosei94@gmail.com](mailto:andrewosei94@gmail.com)

**Comment by RicardoGEsteves** [comment]
Location: Lisbon, Portugal
 Remote: Yes
 Willing to relocate: Yes (Open, based on the country and city in question)
 Technologies (most used lately): 
> - Frontend: `JavaScript`, `TypeScript`, `React`, `Next.js`, `Tailwind`, `Framer Motion`, `Zustand`
> - Backend: `Node.js`, `Express`, `Hono`, `Convex`
> - Database: `PostgreSQL`, `MySQL`, `NoSQL`
>- Testing: `Jest`, `Testing Library`, `Playwright`
> - Other: `Prisma`, `Zod`, `Redux`, `TanStack`, `Server Action`, `Edge Functions`, `tRPC`, `Rest API's`
>
 Resume/CV: [Resume](https://drive.google.com/file/d/1h8nna4uTh3qAdbFUW5BeVqj2RbBCxA2f/view?usp=sharing) 
 Website: https://www.linkedin.com/in/ricardogesteves
 Email: [Email](ricardoesteves.hello@gmail.com)

**Comment by robcipolla** [comment]
Location: UK, Leicester
Remote: Yes
Willing to relocate: No
Technologies:

- **Frontend**: `JavaScript`, `TypeScript`, `React`, `TailwindCSS`, `SCSS`
- **Backend**: `Node.js`, `Express`, `PHP`, `Go (Learning)`
- **Database**: `MySQL`, `PostgreSQL`, `NoSQL`
- **CMS**: `WordPress`, `SanityCMS`, `PayloadCMS`, `StrapiCMS`
- **Other**: `Prisma`, `Zod`, `Redux`, `tRPC` 

Resume/CV: [Resume](https://drive.google.com/file/d/18HFQNdxmHDXBuZQlEO0HyA7b9sxgSw1o/view?usp=sharing)
Website: [Rob Cipolla](https://www.robcipolla.co.uk)
LinkedIn: https://www.linkedin.com/in/rob-cipolla/
Email: hello@robcipolla.co.uk

**Comment by pablojsx** [comment]
Location: Santiago, Chile
Remote: Yes
Willing to relocate: Yes
Technologies: 

**Langs**: `Typescript`, `Python`, `C++`(entry)

**Front**:
- **React**: `Vite`, `Nextjs`
- **CSS**: `Tailwind`(darkmode, responsive), `CSS` for harder tasks, Self experiments.
- **Some FE Libraries**: `React Hook Form`, `React Query`, `NextUI`, `Animate.css`, `React Router`, `Redux`..

**Back**:
- **DB**: `Postgresl`, `MongoDB`
- **ORM/ODM**: `Prisma`, `Mongoose`
- **Emails**: `Sendgrid`
- **Endpoints**: `tRPC`, `Next api routes`, `Expressjs`
- **Some BE Libraries**: `zod-prisma-types`, `Expressjs`, `tRPC`, `Zod`, `Superjson`, `Zod-Fetch`

**Auth**: `Next Auth`, `Express sessions`, `jwt`

**Payment**: `Paypal` with self made types

**Upload**: `Supabase bucket`, `Firebase bucket`

That's all I know

Resume/CV: check website
Website: https://portfoooooooooooolio.vercel.app/
Email: check website
Linkedin: https://t.ly/mdlo1

**Comment by Icegreeen** [comment]
Location: Salvador, BR
Remote: Yes
Willing to relocate: No
Technologies: 

- Language: Typescript | Php | Javascript
- Front-end: React (Next) | Vue (Nuxt) | Tailwind
- Back-end: Node | Laravel 
- Database: MySQL | PostgreSQL
- ORM: Prisma | TypeORM | Eloquent
- CMS: Sanity | Hygraph -> ( GraphQL )

Resume/CV: https://www.linkedin.com/in/flavioaquila/
Website: https://icegreen-dev.vercel.app ⭐
Email: [flavioaquila16@gmail.com](mailto:flavioaquila16@gmail.com)

**Comment by iamarpitpatidar** [comment]
Location: Indore, India
Remote: Yes
Willing to relocate: No
Technologies (most used): 

Frontend: `JavaScript, TypeScript, React, Tailwind`
Backend: `Node.js, PHP (Laravel), Python, Rust`
Database: `MySQL, PostgreSQL, Mongo`
CI/CD: `jUnit, Selenium, Docker, Kubernetes, AWS`

Resume/CV: https://www.linkedin.com/in/arpitpatidar/
Website: https://arpitpatidar.com
Email: arpit.ptdr@hotmail.com

I am only looking for contract work at this time.

**Comment by MZOG** [comment]
Location: **Katowice, Poland**
Remote: **Yes**
Willing to relocate: **No**
Technologies: **React, JavaScript, TypeScript, Tailwind**
Resume/CV: [https://www.linkedin.com/in/marcin-zogrodnik/](https://www.linkedin.com/in/marcin-zogrodnik/)
Website: [marcinzogrodnik.pl](https://marcinzogrodnik.pl)
Email: [kontakt@marcinzogrodnik.pl](mailto:kontakt@marcinzogrodnik.pl)

**Comment by mustafadalga** [comment]
Location: Turkey
Remote: Yes
Willing to relocate: Yes
Technologies: React.js, Vue.js, TypeScript, JavaScript, Next.js, Vuex, Redux Toolkit, Zustand, Webpack, Vite.js, Tailwind CSS, SCSS, CSS3, HTML5, Pinia.
Resume/CV: https://mustafadalga.github.io/cv - https://www.linkedin.com/in/mustafadalga/
Website: https://github.com/mustafadalga/ - https://sft.hashnode.dev/
Email: mustafadalgaa gmail com

Hi 👋,
I am skilled Front-End Developer based in Turkey with 5 years of experience. I am looking a new opportunity.

**Comment by milinddhamu** [comment]
Location: Rajasthan, India
Remote: Yes
Willing to relocate: Yes
Technologies: NextJS, TypeScript, ExpressJS, NodeJS, Tailwind CSS, Styled components, Redux, Recoil, Auth.js, MongoDB, Firebase, GraphQL
Resume/CV: [Resume](https://drive.google.com/file/d/10I5qcErCamhyHFKpKcIsdtUqmy23wbaw/view?usp=sharing)
Website: [Milind Dhamu](https://milinddhamu.com)
Email: milinddhamu6@gmail.com

**Comment by James-4u** [comment]
Location: Toronto, ON, CAN
Remote: Yes
Willing to relocate: No
Technologies: JS/TS, PHP, React/Next.js, Supabase, Firebase, Web3, Solidity, SEO, Performance Optimization
Resume/CV: https://www.linkedin.com/in/jamesjin4u/
Website: https://jamesjin-awesome.netlify.app/
Email: [smart.jamesjin@gmail.com](mailto:smart.jamesjin@gmail.com)
![Portfolio](https://jamesjin-awesome.netlify.app/social-preview.jpg)

**Comment by Althaf-ahammed** [comment]
Location: Kerala, India
Remote: Yes
Willing to relocate: Yes
Technologies: React.js, TypeScript, JavaScript, Next.js, Redux Toolkit, Tailwind CSS, SCSS, CSS3, HTML5, ExpressJS, NodeJS, MongoDB, Jest
Resume/CV: [Resume](https://drive.google.com/file/d/1Wc67pDpoyWMyfF588hkEYaLTFs87qpx9/view?usp=drive_link)
Email:althafahammedm626@gmail.com

**Comment by pavelbuchta** [comment]
Location: Prague, Czechia
Remote: Yes
Willing to relocate: No
Technologies: Typescript/Javascript, MongoDB, Postgres, Prisma, Auth.js, React, Shadcn/ui, Tailwind CSS, React Hook Form
Website: https://exponedesign.com
Email: pavelbuchta80@gmail.com

**Comment by AnggrainiAng** [comment]
Location: Jakarta, Indonesia
Remote: Yes
Willing to relocate: No
Technologies: Javascript, Typescript, React, Redux, Next.js, Tailwind.css, Chakra UI, Boostrap, Docker, MySQL, MongoDB, Mongoose, Git, Github, Prisma, Node.js, Express.js, Postman
Resume/CV: https://www.linkedin.com/in/anggraini-anggraini-5b1771287/
Website: https://stack-overflow-nextjs13-8xmcie9cd-anggrainiangs-projects.vercel.app/
Email: anggi3296@gmail.com

**Comment by MaanilVerma** [comment]
### 👋🏻  **Maanil Verma - Looking for FRONTEND / FULLSTACK ROLES**
___________________________________________________________________________
**Location:** India
**Remote:** Yes
**Willing to relocate:** Yes
**Technologies**: _JavaScript, TypeScript, React JS, Next JS, Node.js, Java, Prisma, Postgres SQL, ChatGPT, MUI, HTML, CSS, StoryBook, Jest & React Testing, GraphQL_

Resume/CV: https://maanilverma.vercel.app/assets/PDF/Software_Engineer_Resume_Maanil_Verma.pdf

Website: https://maanilverma.netlify.app
Email: vermamaanil97@gmail.com
GitHub: https://github.com/MaanilVerma
LinkedIn: https://linkedin.com/in/maanilverma

_____________________________________________________________________________

**Brief ABOUT ME**:

_**Having led the development of 3 products from its inception, one of them achieving 30K LIVE users showcases my ability to deliver impactful results.**_

I bring a proven track record of developing robust software solutions, leveraging my expertise in Java and a range of modern technologies. My proficiency in JavaScript and frameworks like React enables me to create scalable applications with thorough testing practices.

**Comment by AbhiShake1** [comment]
Location: Nepal
Remote: Yes
Willing to relocate: Maybe
Technologies: Flutter, nextjs, tRPC, django
Resume/CV: [Resume](https://github.com/vercel/next.js/files/15211958/Resume__abhi-1.pdf)
Email: abhi.shake.np@gmail.com

**Comment by XoL1507** [comment]
reserved me 1 slot i will hire my teacher.

**Comment by Jaaneek** [comment]
Hi, I have released 4 of my own SaaS and helped companies as a tech lead/consultant, check out my resume and twitter

Location: Poland
Remote: yes
Willing to relocate: yes
Technologies: : React, Next.js, TypeScript, JavaScript, Node.js, tRPC, PostgreSQL, Supabase, Prisma, Drizzle ORM, Tailwind
Resume/CV: https://drive.google.com/file/d/1esSoH6pcNG20a2qcWk-F1QKUq5i5Jo-N/view?usp=sharing
Website: [tryhards.space](https://www.tryhards.space/)
Email: [jankiewiczmilosz.contact@gmail.com](mailto:jankiewiczmilosz.contact@gmail.com)

**Comment by isaacrmoreno** [comment]
Howdy my name is Isaac, get to know me below :) 

Linkedin: https://www.linkedin.com/in/isaacrmoreno/
Website: https://isaacmoreno.vercel.app
Email: keycard@duck.com

Location: Arizona
Remote: Yes
Willing to relocate: No
Frameworks/Libraries: React Native, Expo, Next.js,
Languages: Typescript, Javascript, C#, SQL
Tools/Technologies: Git, Github, DevOps, VSCode, Azure Data Studio, Xcode, Postman
Resume/CV: Happy to share my resume upon request, just email me.

Thank you for taking the time. Cheers!

**Comment by Dannymx** [comment]


**Reply by manovotny** [reply]
@Dannymx just posted them!

**Comment by karomahlailatul** [comment]
Location: Indonesia
Remote: Yes
Willing to relocate: Maybe
Technologies: TypeScript/JavaScript, React, Next.js, Redux, TailwindCSS, Bootstrap, ExpressJS, Fastify, NestJS, Prisma, Drizzle, MySQL, PostgreSQL, MongoDB, Redis, Kafka, RabbitMQ, Go, Gin, Fiber, AWS, Azure, Google Cloud.
Resume/CV: https://docs.google.com/document/d/1e9OmH5rfTOjNnFyBYDqGVCkZzmAUAeCl7pM1b_C7ais/edit?usp=sharing
Website: https://laysdev.com/
Email: [karomahlailatul@gmail.com](mailto:karomahlailatul@gmail.com)

With my experience of around 4 years in software development, I am currently looking for frontend jobs, backend jobs, or full stack javascript.
